app/views/vendors/_details_all_content.haml

```diff
@@ -1,9 +1,8 @@
 #detailsPageContent
-  #leftAlign
+  #leftSidebar
     = image_tag @vendor.picture, id: 'detailsPageImage', class: 'imageWithText'
-    #socialWithText
-      = render 'details_social_buttons'
     #extendTags
       = render 'details_extend_tags'
-  #detailsPageText
-    = render 'details_text'
\ No newline at end of file
+  #detailsPageText.detailsTextWithImage
+    .pageText
```

Comment:
Same comment as on the corresponding product page.

Reply:
See my other comment :)